Source/Scene/MapboxImageryProvider.js

```diff
@@ -0,0 +1,324 @@
+/*global define*/
+define([
+        '../Core/Credit',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/DeveloperError',
+        '../Core/MapboxApi',
+        './UrlTemplateImageryProvider'
+    ], function(
+        Credit,
+        defaultValue,
+        defined,
+        defineProperties,
+        DeveloperError,
+        MapboxApi,
+        UrlTemplateImageryProvider) {
+    "use strict";
+
+    var trailingSlashRegex = /\/$/;
+    var defaultCredit1 = new Credit('© Mapbox © OpenStreetMap', undefined, 'https://www.mapbox.com/about/maps/');
+    var defaultCredit2 = new Credit('Improve this map', undefined, 'https://www.mapbox.com/map-feedback/');
+
+    /**
+     * Provides tiled imagery hosted by Mapbox.
+     *
+     * @alias MapboxImageryProvider
+     * @constructor
+     *
+     * @param {Object} [options] Object with the following properties:
+     * @param {String} [options.url='//api.mapbox.com/v4/'] The Mapbox server url.
+     * @param {String} options.mapId The Mapbox Map ID.
+     * @param {String} [options.accessToken] The public access token for the imagery.
+     * @param {String} [options.format='png'] The format of the image request.
```

Comment:
Am I correct in that this will work with the MapBox formats [supplied here](https://www.mapbox.com/developers/api/maps/#format)?  Can we test it out and be sure.